sample/tests/test_validation.py

```diff
@@ -361,32 +233,12 @@ def test_validation_preserves_pydantic_validation_errors() -> None:
     assert "Quantity must be positive" in str(exc_info.value)
 
 
-def test_temporal_activity_method_validation() -> None:
-    """Test that Temporal activity-decorated methods are handled properly"""
-    mock_minio_payment_repo = MagicMock(spec=MinioPaymentRepository)
-    repo = TemporalMinioPaymentRepository(
-        mock_minio_payment_repo
-    )  # Fixed: only pass payment repo
-
-    # This should pass even though the methods are decorated with
-    # @activity.defn
-    # The validation system should handle Temporal activities gracefully
-    assert isinstance(repo, PaymentRepository)
-
-    # Verify the methods exist and are callable
-    assert hasattr(repo, "process_payment")
-    assert hasattr(repo, "get_payment")
-    assert callable(repo.process_payment)
-    assert callable(repo.get_payment)
-
```

Comment:
Yet another test of `isinstance` with protocols (gotta watch those generative AIs making it harder for us humans to hold context in our heads with cruft code).